### PR TITLE
Fix colors for dropdown menus

### PR DESCRIPTION
**Pull request by esensar**

### CODE REVIEW

**Comment by esensar** [comment]
I am not sure why these dropdown menus don't pick up theme values by themselves. They should behave similarly to dialogs, but I am not sure what I am missing here. It worked after I manually set values in modifier and `Text` composable.

**Comment by tibbi** [comment]
menu colors seem to be good, but the top status and bottom navigation colors seem to be broken on the main screen. Just recheck how should it looks like, both material and non-material themes are wrong.

File: gradle.properties
Comment (FunkyMuse) [inline gradle.properties:19]: Nope, remove this, this is true by default now

**Comment by FunkyMuse** [comment]
why are we migrating the main screen in the samples to Compose, is it necessary?

I think we should leave it for backwards compatibility to check how things "were" and build upon that.

**Comment by esensar** [comment]
> why are we migrating the main screen in the samples to Compose, is it necessary?
> 
> I think we should leave it for backwards compatibility to check how things "were" and build upon that.

I migrated it for easier testing, because I guess compose is the way to go for all the apps. For checking for how things were, I thought we would use the apps that we are migrating to Compose and not main screen from `Commons`.

File: gradle.properties
Comment (esensar) [reply gradle.properties:19]: My bad, I thought I should use `Simple-Thank-You` as reference and it has it set to false.

File: samples/src/main/kotlin/com/simplemobiletools/commons/samples/activities/MainActivity.kt
Comment (FunkyMuse) [inline samples/src/main/kotlin/com/simplemobiletools/commons/samples/activities/MainActivity.kt:55]: update to latest commons please, the name has been changed

File: samples/src/main/kotlin/com/simplemobiletools/commons/samples/activities/MainActivity.kt
Comment (FunkyMuse) [inline samples/src/main/kotlin/com/simplemobiletools/commons/samples/activities/MainActivity.kt:82]: Please extract in a function for better readability

File: samples/src/main/kotlin/com/simplemobiletools/commons/samples/activities/MainActivity.kt
Comment (FunkyMuse) [reply samples/src/main/kotlin/com/simplemobiletools/commons/samples/activities/MainActivity.kt:55]: https://github.com/SimpleMobileTools/Simple-Commons/pull/1879

File: samples/src/main/kotlin/com/simplemobiletools/commons/samples/activities/MainActivity.kt
Comment (FunkyMuse) [inline samples/src/main/kotlin/com/simplemobiletools/commons/samples/activities/MainActivity.kt:55]: it will change to SimpleScaffold soon with the new PR FYI as i think it makes more sense

File: samples/src/main/kotlin/com/simplemobiletools/commons/samples/activities/MainActivity.kt
Comment (esensar) [reply samples/src/main/kotlin/com/simplemobiletools/commons/samples/activities/MainActivity.kt:82]: I will also extract the screen then, to make it a bit more similar to how it should be done in actual apps.

File: samples/src/main/kotlin/com/simplemobiletools/commons/samples/activities/MainActivity.kt
Comment (esensar) [reply samples/src/main/kotlin/com/simplemobiletools/commons/samples/activities/MainActivity.kt]: Updated.

**Comment by FunkyMuse** [comment]
I'll review it today as early as possible as I can

File: commons/src/main/kotlin/com/simplemobiletools/commons/compose/menus/ActionMenu.kt
Comment (FunkyMuse) [inline commons/src/main/kotlin/com/simplemobiletools/commons/compose/menus/ActionMenu.kt:146]: this should be the same as `dialogContainerColor`

**Review by FunkyMuse — APPROVED** [review]
Please change the background color and we're good to go, I have a plane to catch, once you do that change @tibbi please merge it.

**Review by FunkyMuse — APPROVED** [review]

**Comment by FunkyMuse** [comment]
@tibbi we can merge this now

**Pull request merged**
